bin/entry.sh

```diff
@@ -1,8 +1,5 @@
 #!/usr/bin/env sh
 
-bundle exec rails db:create
-bundle exec rails db:migrate
```

Comment:
What happens when someone runs hdm in a container and updates to a newer version? My impression was that the db:migrate task in entry.sh takes care of it so users don't have to deal with it?

Reply:
ah okay, this was the reason why this was like that. got confused by all the other internet examples. will change this again.